### ISSUE TITLE
Edge of straight wall is sinusoidal

### ISSUE THREAD

**Issue by josefnpat** [josefnpat/VividCast#14]
![image](https://user-images.githubusercontent.com/646162/32386373-fa5cadfe-c08e-11e7-9b5b-905d23e8d133.png)

@Positive07 what do you think?

**Comment by pablomayobre** [comment]
Yeah, I saw that too, and is probably related to #12 but it will be harder to fix that, since it's probably due to an error in the calculation of the ray length
